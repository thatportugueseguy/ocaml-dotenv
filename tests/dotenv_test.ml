let () =
  let env_variables = Dotenv.parse () in
  List.iter (fun (name, value) -> print_endline (Printf.sprintf "%s=%s" name value)) env_variables
