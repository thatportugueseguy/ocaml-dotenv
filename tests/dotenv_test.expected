HELLO=World
DEBUG=true
MULTISPACE=a a a a
WORKS=
JSON={"foo": "bar"}
WHITESPACE=nope
SINGLE_QUOTE="quoted"
VAR="    some value "
